blog-server-api/src/endpoints/create_post/handler.rs

```diff
@@ -0,0 +1,41 @@
+use super::request_content::{CreatePostRequestContent, CreatePostRequestData};
+use super::response_content_failure::CreatePostContentFailure;
+use super::response_content_failure::CreatePostContentFailure::*;
+use super::response_content_success::CreatePostContentSuccess;
+use crate::extensions::Resolve;
+use blog_server_services::traits::post_service::PostService;
+use screw_api::request::ApiRequest;
+use screw_api::response::ApiResponse;
+use screw_components::dyn_result::DResult;
+use std::sync::Arc;
+
+async fn handler(
+    new_post_data: DResult<CreatePostRequestData>,
+    post_service: Arc<Box<dyn PostService>>,
+) -> Result<CreatePostContentSuccess, CreatePostContentFailure> {
+    let CreatePostRequestData {
+        post_id,
+        post_content,
+    } = new_post_data.map_err(|e| ValidationError {
+        reason: e.to_string(),
+    })?;
+
+    let test = post_service
```

Comment:
xyest

Reply:
Inappropriate communication is unprofessional and goes against our corporate standards. Let's maintain a respectful and professional environment for everyone's benefit. #ProfessionalismMatters

Reply:
was needed for testing purposes, now removed